x/upgrade/keeper/keeper.go

```diff
@@ -191,7 +237,7 @@ func (k Keeper) ApplyUpgrade(ctx sdk.Context, plan types.Plan) {
 		panic("ApplyUpgrade should never be called without first checking HasHandler")
 	}
 
-	handler(ctx, plan)
+	handler(ctx, plan, k.GetConsensusVersions(ctx))
```

Comment:
we should handle the error here too

Reply:
[fixed](https://github.com/cosmos/cosmos-sdk/pull/8743/commits/036db4f5c1ae7c9709ff2373fa57cb1460d9f370#diff-33ff09992f1a3583e2aa8eb932c62c833d52e446488070ed958eb672eed8da3e)